foxsi.pro

```diff
@@ -0,0 +1,42 @@
+; June 2016
+; STARTUP FOXSI
+; Description: 
+;
+;
+
+PRO FOXSI, YEAR
+
+COMMON FOXSI
```

Comment:
I figured that it may be a way around it that involves using '.sav' files, but I already read that you don't like that option either. Do you have any advice?